libs/build.js

```diff
@@ -0,0 +1,133 @@
+const path = require('path');
+const utils = require('./utils');
+const template = require('./template');
+const memo = require('./memo').memo;
+
+const DEFAULT_IGNITOR_EVENT = {
+  "ignitor": true,
+}
+
+const DEFAULT_FUNCTION_OPTIONS = {
+  name: '.*',
+  schedule: true,
+}
+
+const DEFAULT_IGNITOR_SCHEDULE = {
+  rate: 'rate(5 minutes)',
+  enabled: true,
+  input: DEFAULT_IGNITOR_EVENT
+};
+
+const DEFAULT_OPTIONS = {
+  functions: [
+    DEFAULT_FUNCTION_OPTIONS,
+  ],
+};
+
+// !!WARNING!! Do not include a '.' in the directory name 
+const PACKAGE_DIR = 'ignitor';
+
+const resolveWrapper = (servicePath, wrapper) => {
+  if (wrapper) {
+    const filename = path.join(servicePath, wrapper);
+    return utils.read(filename);
+  }
+  return null;
+};
+
+const buildRegex = (name) => {
+  const regexsplit = name.split(/(?<!\\)\//);
+  if (regexsplit.length === 1) {
+    return new RegExp(name);
+  }
+  const [ignore, regExp, flags] = regexsplit;
```

Comment:
you can remove `ignore`. Ex: [, regExp, flags]`

Reply:
oh really, didn't know that, nice